engines/director/lingo/lingo-object.h

```diff
@@ -205,11 +213,12 @@ class ScriptContext : public Object<ScriptContext> {
 	SymbolHash _functionHandlers;
 	Common::HashMap<uint32, Symbol> _eventHandlers;
 	Common::Array<Datum> _constants;
-	DatumHash _properties;
```

Comment:
This breaks the current master, where this hash is accessed by the visual debugger

Reply:
Fixed